### DISCUSSION TITLE
Input Leap released for MacOS, do you guys have any idea when we will get for Windows and Linux?

### DISCUSSION THREAD

**Discussion by henriquebach** [input-leap/input-leap#1989] [Development of Input Leap]
First, congratulations for the release! You guys pick up from where Barrier left and finaly did it!

I notice only .dmg for MacOS is released, do you guys have any idea when we will get Windows and Linux builds? In my case I want to use Windows machine as server and Linux Wayland as client.

Thank you guys!

**Comment by Shuna322** [comment]
For Linux, give Flatpak a try. The CI automatically builds the Flatpak package, check the release job for an artifact, and install the package from the file.
I'm personally using the Flatpak variant, tested on X11 and Wayland (Ubuntu 24.04 LTS)
As for Windows - Similar build packages are generated, even with installers, but those installers don't ship the QT dll's thus the applications won't start. I guess the build process could be improved here 😉
![image](https://github.com/user-attachments/assets/418573ea-5eb9-416c-8524-d19860b4ef9c)

**Reply by henriquebach** [reply]
No problem with Flatpak, but I don't want to have to build it. I want the binaries like the MacOS release, preferable Flatpak for Linux, and .exe ou .msi for Windows.

**Reply by ViRGEdx** [reply]
> No problem with Flatpak, but I don't want to have to build it. I want the binaries like the MacOS release, preferable Flatpak for Linux, and .exe ou .msi for Windows.

For what it's worth, if you want a working build to play with for now, the Windows QT6 (2022) package from the following CI build is working for me: https://github.com/input-leap/input-leap/actions/runs/11172901538

The necessary QT widgets are included. So it runs out of the box.

**Reply by gitfmc** [reply]
@ViRGEdx but how do you install it? The link you provided only gives a zip file, inside it has another zip file named debug, and inside a bunch of files. How do you install those? It would be much simpler to just have an .exe file. Is this not possible to create?

**Reply by ViRGEdx** [reply]
> @ViRGEdx but how do you install it? The link you provided only gives a zip file, inside it has another zip file named debug, and inside a bunch of files. How do you install those? It would be much simpler to just have an .exe file. Is this not possible to create?

It's a portable build that doesn't require installation. Just unpack the debug.zip file to a directory of your choice, and then run input-leap.exe

As the devs have noted elsewhere, this isn't how the final release version of Input-Leap will be distributed. But it's a working, full-fledged version of 3.0.0 that you can tinker with for now until a proper release is packaged.

**Comment by gitfmc** [comment]
I'm here to second the OP, both on the congratulations on the first release and request for proper windows and Linux installers.
I've been waiting for years for this release, so I'm a bit disappointed I won't be able to use it still. Barrier is getting too buggy on Windows, I'l almost considering giving synergy another try.
Thank you for the awesome work though, and I look forward to the release of Windows and Linux working installers.

**Reply by henriquebach** [reply]
I agree but also they are doing this voluntary, so we can only thank them for continue the work on barrier. But yeah please Windows and Linux installers.

**Reply by gitfmc** [reply]
@henriquebach absolutely, and I dont want to sound ungrateful. I thank the devs taking over~the project with all my heart as synergy, then barrier, and hopefully input-leap are an important part of my workflow. I've just been checking in on the project for a couple years, always hoping for a release and just disapointed the .exe and linux installers weren´t release. But thank you to the devs for taking on this prroject and. as I said, congrats on the first release after this couple of years. Wish I could give it a try.

**Comment by p12tic** [comment]
Windows and Linux builds will be added in a new release over the next 2 days.

**Reply by henriquebach** [reply]
Great! Thank you guys.

**Reply by firsttiger** [reply]
Can't wait to try it out!

**Reply by delath** [reply]
Looking forward to it 😉

**Comment by p12tic** [comment]
InputLeap has been released for Windows and Linux. On Ubuntu and Debian Wayland is not enabled yet, this will be done in next release in the following days. I'm thus closing the issue.
